styles/style.css

```diff
@@ -6,3 +6,143 @@ Green: #00B172
 White: #FFF
 
 */
+body{
+    font-family: 'Montserrat', sans-serif;
+}
+nav {
+    padding: 0 25px;
+    display: flex;
+    justify-content: space-between;
+   
+  }
+  
+  nav ul li{
+      margin: 6px;
+  }
+  ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-around;
+  }
+  
+nav a {
+padding: 0.25px;
+    text-decoration: none;
+    color: black;
+}
+.spLogo{
+    width: 150px;
+}
+#text{
+    background-image: url("../images/landing.jpg");
+    height: 600px;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    color: white;
+    text-align: center;
+    justify-content: center;
+}
+#text p {
+    text-align: center;
+}
+.green-section{
+    background-color:  #00B172 ;
+    color: #FFF;
+    padding: 3vh;
+    position: relative;
+    padding: 6vh;
+}
+.green-section > h4 {
+    width: 69vh;
+    border-bottom: 3px solid white;
+    font-size: 5vh;
+    font-weight: 500;
+    padding-bottom: 2vh;
+    margin-block-start: 0vh;
+}
+.display{
+    display: flex;
+    width: 90%;
+    margin: 0 auto;
+    justify-content: space-between;
+    align-items: center;
+}
+.article{
+    width: 30%;
+}
+.article > img{
+    width: 40vh;
+}
+.section > h5 {
+    margin-block-start: 1em;
+    font-size: 3vh;
+    margin-block-end: 1em;
+}
+
+.section > p {
+    font-size: 2vh;
+    width: 30vh;
+}
+.logo > img {
+    width: 15vh;
+    position: absolute;
+    top: 40vh;
+    left: 90vh;
+}
```

Comment:
In this section, you could have a different HTML structure, and a cleaner CSS if you applied the logo as `background-image` at the `.green-section` div.

`
  background: url("../images/spotify-icon-white.png") no-repeat;
  background-position: center;
  background-color: #00b172;
  background-size: 130px;
`